### ISSUE TITLE
Account Picture not defined in IDL and no semantics provided

### ISSUE THREAD

**Issue by bvandersloot-mozilla** [fedidcg/FedCM#406]
Speaking of the following struct: https://fedidcg.github.io/FedCM/#dictdef-identityprovideraccount.

The semantics of the picture variable are missing and are not included in the IDL.

**Comment by bvandersloot-mozilla** [comment]
Yeah, I was surprised to see I hadn't added it with the IDL and then noticed it wasn't in the list below. 

I'd file a patch to fix this, but I don't know what your rules on the URL are (absolute vs relative in particular).

**Comment by npm1** [comment]
I sent a patch though waiting for Christian to check it :) AFAICT it requires absolute URLs and the fetch is done from the broweser, i.e. there is no referer etc. I do not see us setting the request's destination though.
